bibcheck/plugins/fix_ref_jhep_volume.py

```diff
@@ -550,6 +550,11 @@ def get_pubnote_from_inspire(ref_pbn, bug_type, recid, debug=False):
 
     return pubnote
 
+def add_noncurator(ref_subfields):
+    if not ('9','NONCURATOR') in ref_subfields:
```

Comment:
could there be anything else besides (NON-)CURATOR in subfield 9? that would lead to multiple subfields 9. is that allowed?

Reply:
I was mainly concerned about conflicting information. The answer is yes, but I guess the other values are not intentionally:
```
In [4]: f = get_all_field_values('999C59')

In [6]: f
['CURATOR', '1977', 'CURATOR$o9', 'DESY', 'CURATOR$rarXiv:1206.3013', 'refextract', '1570', 'CURATOR$hPraton E.A., Melott A.L., McKee M.Q.', '2009', '580997', 'script', '1984', '1987', '1982', 'CURATOR$hHeisenberg W and Euler H', 'refextract, P. Evans and P. Hudson', 'CURATORG. V. Efimov and M. A. Ivanov', 'CURATOR$hG. BOILLAT', 'CURATOR$hL. Brun', 'CURATOR 999C6', 'CURATOR$hG. V. Efimov and M. A. Ivanov', 'Entanglement entropies of the quarter filled Hubbard model', 'CURATOR$h$P.W. Milonni', 'CURATOR$hV.M. Mostepanenko, N.N. Trunov', 'CURATORHeisenberg W and Euler H', 'CURATORK\xc5\x82usek-Gawenda M, Lebiedowicz P and Szczurek A', 'CURATOR$hWith V.N. Gribov', 'CURATOR5', '279039', '66170']
```
apart from [script](http://old.inspirehep.net/search?ln=en&p=999C59%3Ascript&of=hb&action_search=Search&sf=earliestdate&so=d)

Reply:
yeah, mostly mistakes. I cleaned those up. there are two very frequent ones, though
```ipython
In [8]: for c in get_all_field_values('999C59'):
    x = perform_request_search(p="999c59:%s" % c)
    if x:
        print c, x[:5], len(x)
   ...:         
CURATOR [1, 113, 616, 660, 693] 118653
refextract [615, 1209, 1521, 2313, 2693] 109289
script [807017, 813002, 834354, 837451, 848544] 988
```

this makes me think that `NONCURATOR` may as well be `script` ?